main/2021/CG-08-03.md

```diff
@@ -27,7 +27,14 @@ Installation is required, see the calendar invite.
     1. Review of action items from prior meeting.
     1. Update on [Branch Hinting](https://github.com/WebAssembly/branch-hinting) [20 min].
         1. Poll for phase 3
-    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [30 min].
+    1. Summary of [interface-types/#135](https://github.com/WebAssembly/interface-types/issues/135) and discussion [40 min].
+        1. Poll for supporting [UTF-16/Latin1](https://github.com/WebAssembly/interface-types/issues/136#issuecomment-861799460) lifting and lowering
```

Comment:
I feel we should poll for UTF-16 and UTF-16/Latin1 separately.

Reply:
Sure, I can split this one :)